packages/esm-patient-list-management-app/src/hooks/usePatientListing.tsx

```diff
@@ -246,164 +227,41 @@ export const usePatientListing = () => {
     }
     if (currentPaginationState.page > 0) {
       if(currentTab === 0) {
-        getAllClients({
+        getClients({
```

Comment:
@Samstar10 This whole block can be refactored further
`if(currentTab > 0 && currentTab < 5){
  getClients({
          currentPage: currentPaginationState.page,
          pageSize: currentPaginationState.size,
        })
}`